UI/Source/CollectionViews/CollectionViewModelDataSource.swift

```diff
@@ -1062,15 +1057,10 @@ extension CollectionViewModelDataSource: NSCollectionViewDataSource {
 
     private func rebindViewAtIndexPath(_ indexPath: IndexPath, toViewModel viewModel: ViewModel) {
         guard let collectionView = collectionView else { return }
-        guard let item = collectionView.item(at: indexPath) as? CollectionViewHostItem else { return }
+        guard let item = collectionView.item(at: indexPath as IndexPath) as? CollectionViewHostItem else { return }
```

Comment:
That was just part of the revert of the previous commit, will remove though since 8.2 seems ambivalent towards it.